pandas/core/generic.py

```diff
@@ -4104,6 +4105,22 @@ def isnull(self):
     def notnull(self):
         return notnull(self).__finalize__(self)
 
+    def _clip_with_scalar(self, lower, upper):
+
+        if ((lower is not None and np.any(isnull(lower))) or
+                (upper is not None and np.any(isnull(upper)))):
```

Comment:
Are the `np.any` needed here? As lower/upper are already confirmed to be a scalar?